std/uni.d

```diff
@@ -7289,6 +7290,19 @@ if (isInputRange!Range && is(Unqual!(ElementType!Range) == Grapheme))
 Range byCodePoint(Range)(Range range)
 if (isInputRange!Range && is(Unqual!(ElementType!Range) == dchar))
 {
+    static struct Result(R)
```

Comment:
Am I missing something, because it looks like this struct isn't used anywhere?

Reply:
Omg. It's good we have reviews though.